shared/wallets/transaction-details/index.js

```diff
@@ -141,17 +143,27 @@ const propsToParties = (props: NotLoadingProps) => {
   const yourAccountID = props.yourRole === 'senderOnly' ? props.senderAccountID : props.recipientAccountID
   const counterpartyAccountID =
     props.yourRole === 'senderOnly' ? props.recipientAccountID : props.senderAccountID
-  const you = (
-    <NameWithIcon
-      colorFollowing={true}
-      horizontal={true}
-      onClick={() => props.onShowProfile(props.you)}
-      underline={true}
-      username={props.you}
-      metaOne="You"
-      metaTwo={yourAccountID ? <SmallAccountID accountID={yourAccountID} /> : null}
-    />
-  )
+  const you =
+    props.counterpartyType === 'otherAccount' && props.youWereWallet ? (
```

Comment:
`CounterParty` already has switching between showing `NameWithIcon` vs `CounterPartyText` etc. I think that should be making this decision too.

Reply:
`CounterPartyText` and `CounterPartyIcon` are used a lot, `CounterParty` is only in this file though